### PR TITLE
TH-987: filter out espoo events

### PR DESCRIPTION
**Pull request by nenonja**
## Description :sparkles:
Events search page still shows espoo events when the page is opened initially. Let's fix this by using new fixed parameter value: `all_ongoing=true`
- parameter added to search
- refactored test code: eventListQuery mocks are done in one place
- found one extra bug while refactoring, OtherEventTimes query used `startDate` parameter which does not exist. Fixed + changed the code so that typescript will catch it now.
- refactored test code: move all test utils to `src/test/` folder 
- Pr on events-helsinki-api-proxy: https://github.com/City-of-Helsinki/events-helsinki-api-proxy/pull/51

NOTE: refactoring is in separate commits, so it might be easier to review this pr commit by commit

## Issues :bug:
https://helsinkisolutionoffice.atlassian.net/browse/TH-987

## Testing :alembic:
https://github.com/City-of-Helsinki/events-helsinki-api-proxy/pull/51 must be merged first to get test environment, or locally checkout both prs and generate `yarn codegen` on both services.

## Additional notes :spiral_notepad:

### CODE REVIEW

**Comment by nenonja** [comment]
It seems that browser-tests nicely catch regression bug, click more events button has stopped working:
```
 ✖ "click more events" -button works (screenshots:
 /home/runner/work/events-helsinki-ui/events-helsinki-ui/screenshots/2021-03-01_08-18-45/test-3/Chrome_88.0.4324.182_Linux_0.0/errors/1.png)

   1) Error: Unknown argument "allOngoing" on field "eventList" of type
      "Query". Did you mean "allOngoingOr", "allOngoingAnd",
      "localOngoingOr", or "localOngoingAnd"?:
      {"response":{"errors":[{"message":"Unknown argument \"allOngoing\"
      on field \"eventList\" of type \"Query\". Did you mean
      \"allOngoingOr\", \"allOngoingAnd\", \"localOngoingOr\", or

   \"localOngoingAnd\"?","locations":[{"line":2,"column":13}],"extensions":{"code":"GRAPHQL_VALIDATION_FAILED","exception":{"stacktrace":["GraphQLError:
      Unknown argument \"allOngoing\" on field \"eventList\" of type
      \"Query\". Did you mean \"allOngoingOr\", \"allOngoingAnd\",
      \"localOngoingOr\", or \"localOngoingAnd\"?","    at Object.Argument
```

Need to address this.

**Review by Sartonon — APPROVED** [review]
Looking very good. Excellent improvement!

File: browser-tests/common.components.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ export const getCommonComponents = (t: TestController) => {
   const loadingSpinner = () => {
     const selectors = {
       spinner() {
-        return screen.findByTestId('loading-spinner');
+        return screen.queryAllByRole('loading-spinner');
```
Comment (Sartonon) [inline browser-tests/common.components.ts:11]: How can this work? `queryAllByRole`?

Comment (nenonja) [reply browser-tests/common.components.ts:11]: copy-paste error

**Pull request merged**